Source/JavaScriptCore/llint/InPlaceInterpreter.asm

```diff
@@ -378,6 +378,11 @@ macro instructionLabel(instrname)
     _ipint%instrname%_validate:
 end
 
+macro slowPathLabel(instrname)
```

Comment:
Does a slow path need to be aligned?

Reply:
Pretty sure it doesn't, since we never dispatch to it, it's just part of the normal `local.get/set/tee` implementation we're moving out-of-line since it's a rare case.